packages/trader/src/AppV2/Components/TradeParameters/Duration/container.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from 'react';
+import { ActionSheet, CaptionText, Text } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+import { useTraderStore } from 'Stores/useTraderStores';
+import DurationEndTimePicker from './datepicker';
+import { observer } from '@deriv/stores';
+import DurationChips from './chips';
+import DurationWheelPicker from './wheelpicker';
+
+const DurationActionSheetContainer = observer(
+    ({ selected_hour, setSelectedHour }: { selected_hour: number[]; setSelectedHour: (arg: number[]) => void }) => {
+        const { duration, duration_unit, duration_units_list, onChangeMultiple, expiry_time } = useTraderStore();
+        const [unit, setUnit] = useState(expiry_time ? 'et' : duration_unit);
+        const [selected_time, setSelectedTime] = useState([duration]);
+        const [expiry_date_data, setExpiryDate] = useState<Date>(new Date());
+        const [end_time, setEndTime] = useState<string>('');
```

Comment:
We can omit <string> here and <boolean> below, but it's absolutely optional, so up to you